src/Controller/Component/FormProtectionComponent.php

```diff
@@ -87,9 +87,9 @@ protected function _getSessionId(): string
      * Token check happens here.
      *
      * @param \Cake\Event\EventInterface<\Cake\Controller\Controller> $event An Event instance
-     * @return \Cake\Http\Response|null
+     * @return null
      */
-    public function startup(EventInterface $event): ?Response
+    public function startup(EventInterface $event): null
```

Comment:
You can keep the return type to be the same, in bugfix versions we shouldnt change the signature.
We just make sure not to return anything else than null and are good.

Reply:
changed it back